pyproject.toml

```diff
@@ -39,6 +39,8 @@ exclude = ["tests", "docs"]
 [tool.poetry.dependencies]
 python = ">=3.6.2"
 django = ">=2.2"
+dj-database-url = "^1.0.0"
```

Comment:
Are you sure this should be here?

Reply:
good point, this is a dependency for the test app, not for jazzmin itself, you can add it into the dev dependencies if you like, though the test app dependencies are tracked here https://github.com/farridav/django-jazzmin/blob/master/tests/test_app/requirements.txt so they can get picked up by the heroku deploy